app/companies/create-job/page.tsx

```diff
@@ -5,13 +5,21 @@ import { useState, FormEvent } from "react";
 import Autosuggest from "react-autosuggest";
 import toast from "react-hot-toast";
 
+import { SelectInput } from "../../components/select-input";
 // TODO: use button but before add the type of the button component (i.e. type="button" or type="submit")
 // import { Button } from "../../components/button";
 import { skills } from "../../constants/skills";
+import { currencies } from "@/app/constants/currencies";
+
+interface Option {
+  value: string;
+  label: string;
+}
```

Comment:
Use LabelOption from @interfaces

Reply:
See https://github.com/GoodHive/app/pull/103 (named it Option instead of LabelOption)